### PR TITLE
feat(web): Organization page sidebar card update

### PR DESCRIPTION
**Pull request by RunarVestmann**
# Organization page sidebar card update

## What

* Added support for mailing list signup in sidebar of organization pages
* Added support for connected components in sidebar of organization pages (now we're using a connected component to render a newscard for the latest news of an organization)
* I added box props to the newsletter signup component to be able to make the content fit inside the sidebar
* Removed default_with_image theme from contentful types since it's not used and was updated in Contentful

## Why

* This was requested by Stafrænt Ísland

## Screenshots / Gifs

image
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx [hunk 1]
```diff
@@ -434,18 +439,76 @@ export const OrganizationWrapper: React.FC<WrapperProps> = ({
...
+                            inputAndButtonContainerBoxProps={{
+                              display: 'flex',
+                              flexDirection: 'column',
+                              alignItems: 'flexStart',
+                            }}
+                            buttonBoxProps={{
+                              marginTop: 3,
+                            }}
+                            variant="white"
+                            namespace={namespace ?? (({} as unknown) as any)}
```
Comment (Ingimarsson) [inline apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx:504]: Is there no better type that can be used?

File: libs/cms/src/lib/models/sidebarCard.model.ts [hunk 2]
```diff
@@ -24,11 +25,15 @@ export class SidebarCard {
...
+export const mapSidebarCard = ({
+  sys,
+  fields,
+}: ISidebarCard): SystemMetadata<SidebarCard> => ({
+  typename: 'SidebarCard',
   id: sys.id,
   type: fields.type ?? '',
   title: fields.title ?? '',
-  content: fields.content ?? '',
+  contentString: fields.content ?? '',
```
Comment (Ingimarsson) [inline libs/cms/src/lib/models/sidebarCard.model.ts:36]: Any reason for this?

**Review by Ingimarsson — APPROVED** [review]

File: libs/cms/src/lib/models/sidebarCard.model.ts (hunk 2 above)
Comment (RunarVestmann) [reply libs/cms/src/lib/models/sidebarCard.model.ts:36]: Yes, we can't have two fields with the same name in fragments.ts sadly so I had to rename this

File: apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx (hunk 1 above)
Comment (RunarVestmann) [reply apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx]: I changed the namespace type so this has been addressed :)

File: apps/web/components/MailingListSignup/MailingListSignup.tsx [hunk 3]
```diff
@@ -21,7 +20,7 @@ interface FormProps {
 }
 
 interface MailingListSignupProps {
-  namespace: GetNamespaceQuery['getNamespace']
+  namespace: Record<string, string>
```
Comment (kksteini) [inline apps/web/components/MailingListSignup/MailingListSignup.tsx:23]: Hmmm... are the generated types failing?
I'd be interested to know how

Comment (RunarVestmann) [reply apps/web/components/MailingListSignup/MailingListSignup.tsx:23]: We are JSON parsing the object before it is passed in as props so the typing is wrong. I could've changed this in another PR but I changed it here.

**Review by kksteini — APPROVED** [review]

**Pull request merged**